letsdrill.rb

```diff
@@ -1,19 +1,28 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case integer
+  when 91..100
+    "A"
+  when 81..90
+    "B"
+  when 71..80
+    "C"
+  when 61..70
+    "D"
+  when integer<60
+    "F"
+  else
+    "Invalid Input"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort_by {|string| string.length}[0]
```

Comment:
Nice.  Ruby has `first`, so that is typically preferred to `[0]` for stylistic reasons.  But also, scan through the ruby [Enumerable documentation](https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-min_by), and see if you can't find a method that is even more fitting than what you have here.